app/Http/Controllers/Adm/Mship/Feedback/FeedbackSendController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Controllers\Adm\Mship\Feedback;
+
+use App\Models\Mship\Feedback\Feedback as FeedbackModel;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Redirect;
+
+class FeedbackSendController extends \App\Http\Controllers\BaseController
+{
+    public function store(FeedbackModel $feedback, Request $request)
+    {
+        $feedback->markSent(\Auth::user(), $request->input('comment'));
```

Comment:
Might as well use the `use Illuminate\Support\Facades\Auth;` here

Reply:
f5c3990